cmake/Modules/EttercapLibCheck.cmake

```diff
@@ -117,22 +117,18 @@ endif(HAVE_ICONV)
 
 # LTDL
```

Comment:
On FeeBSD for example this variable would be empty because dlopen is part of libc. Your latest approach would disable plugin support on FreeBSD at least.